packages/babel-helper-create-class-features-plugin/src/index.ts

```diff
@@ -266,17 +259,6 @@ export function createClassFeaturePlugin({
         }
       },
 
-      PrivateName(path: NodePath<t.PrivateName>) {
-        if (
-          this.file.get(versionKey) !== version ||
-          path.parentPath.isPrivate({ key: path.node })
-        ) {
-          return;
-        }
-
-        throw path.buildCodeFrameError(`Unknown PrivateName "${path}"`);
```

Comment:
This check was causing problems because if we only enable the private methods plugin, some private names might not be transformed (i.e. if they refer to a private field).
We now also validate private names usage in the parser, so it's safe to remove it from here.